backtester/main.go

```diff
@@ -203,6 +222,12 @@ func main() {
 		log.Info(log.GRPCSys, "Ready to receive commands")
 	}(btCfg)
 	interrupt := signaler.WaitForInterrupt()
+	if btCfg.StopAllJobsOnClose {
+		_, err = runManager.StopAllTasks()
```

Comment:
Suggestion: Could be fancy and print out what's been stopped. Not really a priority though.